src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarExtraLinks.tsx

```diff
@@ -1,4 +1,6 @@
-import { Box, Link, Sans, Separator, Spacer } from "@artsy/palette"
+// FIXME -- Remove this lint disable
+/* eslint-disable jsx-a11y/anchor-is-valid */
```

Comment:
Yeah this one looks more complicated and can/should be addressed in a separate PR. For actual anchor tags that *also* have `onClick` handlers you want to: tabIndex={0} to make them focusable, then add a keypress handler to trigger the same onClick action with enter/spacebar.